src/include/btree.i

```diff
@@ -1543,6 +1545,48 @@ __wt_btree_lsm_over_size(WT_SESSION_IMPL *session, uint64_t maxsize)
 }
 
 /*
+ * __wt_btree_lsm_flip_primary --
```

Comment:
`s/flip/switch/g` for consistency with LSM terminology, where we "switch" in a new primary.